docusaurus/docs/introduction/plugin-types-usage

```diff
@@ -0,0 +1,133 @@
+
+::: 
+
+### Provisioning of apps
+
+Use the [Grafana configuration file](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/) to configure your app plugins.
+
+Apps can also be provisioned using Grafana’s [provisioning features](https://grafana.com/docs/grafana/latest/administration/provisioning/) so that you can use YAML files to start them when Grafana starts or on demand. Note that the plugin must be installed before provisioning can succeed with a `plugin id` of `app` type.
+
+### Bundling of apps
+
+The app plugin type allows you to [nest other plugins inside it](../create-a-plugin/extend-a-plugin/nested-plugins); in other words, to bundle or distribute multiple plugins in the same package. 
+
+### Bundling of dashboards
+
+App plugins can include dashboards by referencing dashboard JSON files within the `plugin.json` including `property` and `type=dashboard`. Grafana puts an app plugin in the `General` folder when it is imported.
+
+## Panel (visualization) plugins
+
+Development of panel plugins, also known as visualizations, allows Grafana to use custom visualizations, similar to existing plugins such the [Worldmap](https://grafana.com/grafana/plugins/grafana-worldmap-panel/) panel. 
+
+### Usage of panel plugins
+
+Any installed plugin of type `panel` can be used, selected, and configured as a visualization within a dashboard.
+
+Use the plugins API ([`/api/plugins`](https://grafana.com/api/plugins), `/api/plugins/\<plugin id\>/`) for endpoints to retrieve installed plugins, including panel plugins. 
+
+Compared with data sources and apps, there’s currently no support for configuring visualizations on a [Grafana organization](https://grafana.com/docs/grafana/latest/administration/organization-management/#about-organizations) level. 
+
+:::note
+
+With the recent introduction of [Grafana Scenes](https://grafana.com/developers/scenes), you can use any installed plugin of type `panel` as a visualization within a Scenes implementation.
+
+:::
+
+### Provisioning of panels
+
+Use the [Grafana configuration file](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/) to configure your app plugins.
+
+## Server-side image rendering plugins
+
+Rendering plugins allows the Grafana server to render images. Examples of plugins using image rendering include [Grafana Reporting](https://grafana.com/docs/grafana/latest/dashboards/create-reports/) and [Grafana Alerting](https://grafana.com/docs/grafana/latest/alerting/). 
+
+### Usage of rendering plugins
+
+Grafana does not currently provide SDK support for implementing rendering plugins.
+
+## Secrets management plugins
+
+Plugins can also be used to manage encrypted fields to which only the backend server has access, such as API keys or certifications.
+
+### Usage of secrets management plugins
+
+Grafana does not currently provide SDK support for implementing secrets management plugins.
```

Comment:
As discussed earlier we can skip this since not officially supported

Reply:
done.